main.py

```diff
@@ -1,9 +1,11 @@
+from datetime import datetime
 import os
 import sys
 import logging
 import platform
 import subprocess
 import tempfile
+import csv
```

Comment:
This is not consistent with the other parts of the code. For instance you've used pandas for importing the JPeek tool output. Why using csv now?